css-in-javascript/README.md

```diff
@@ -19,7 +19,7 @@
     ```js
     // bad
     {
-      'bermuda-triangle': {
+      bermuda-triangle: {
```

Comment:
This change actually makes it incorrect; since in JS, `bermuda-triangle` is an invalid identifier, and thus it requires the quotes.